release-team/role-handbooks/ci-signal/template-ci-signal-shadow-onboarding-agenda.md

```diff
@@ -0,0 +1,87 @@
+# 20XX-XX-XX - Kubernetes Release X.XX CI Signal Shadow Onboarding
+
+Meet the Team
+
+| Name  | Slack | GitHub | K8s since | Timezone |
+| ---   | ---   | ---    | ---       |  ---     |
+| ...   | ...   | ...    | ...       | ...      |
+
+Timezones and possible meeting slots
+
+| Possible meeting slot  | Timezone A | Timezone B | Timezone C |
+| ------------- | ------------- |------------- |------------- |
+| < ✅, ⚠️, ❌ >  | 6 am  | 9 pm  | 2 pm  |
+
+
+## Responsibilities
+
+The scope of CI Signal team is to monitor the results from our CI and raise any issues we find. 
+
+Job comes down to:
+
+
+1. Monitor testgrid: Particularly interested the [testgrid dashboards](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#blocking-vs-informing-dashboards): [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) & [master-informing](https://testgrid.k8s.io/sig-release-master-informing).
```

Comment:
this part is redundant to another PR https://github.com/kubernetes/sig-release/pull/1869
as well as the [overview](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/ci-signal#overview-of-ci-signal-responsibilities) section of the handbook

Reply:
yes! I think we can reference it as a place holder so we have a single source of truth!

Reply:
yes! I think we can reference it as a place holder so we have a single source of truth!